Source/IndexDefines.H

```diff
@@ -58,11 +58,22 @@
 
 #define UFS (UFA + NUM_ADV)
 #define UFX (UFS + NUM_SPECIES)
-#define NVAR (NTHERM + NUM_ADV + NUM_SPECIES + NUM_AUX)
+#define ULIN (UFX + NUM_AUX)
```

Comment:
Can you add documentation for these? Kinda like we do for `NUM_ADV` in the Docs?